stablehlo/tests/CMakeLists.txt

```diff
@@ -26,7 +26,6 @@ add_lit_testsuite(check-stablehlo-tests "Running the tests/ suite"
   stablehlo-opt
   stablehlo-translate
 )
-add_dependencies(check-stablehlo check-stablehlo-tests)
```

Comment:
This is probably unintended? I think we'd still want to run `check-stablehlo-tests` as part of `check-stablehlo`.